pkg/sql/logictest/tests/local-vec-off/local-vec-off_test: TestLogic_fk_subset_unique failed

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://mesolite.cluster.engflow.com/invocations/default/a13e3d9d-0037-44bc-9c2a-c430fa71f872?testReportRun=1&testReportShard=25&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy9sb2NhbC12ZWMtb2ZmOmxvY2FsLXZlYy1vZmZfdGVzdA==) on master @ [1fcef18003b8dec750ed636aa92d35ca10105878](https://github.com/cockroachdb/cockroach/commits/1fcef18003b8dec750ed636aa92d35ca10105878):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: outputs.zip/logTestLogic_fk_subset_unique1578831332
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[15:39:50] --- done: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[15:39:50] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: outputs.zip/logTestLogic_fk_subset_unique1578831332
--- FAIL: TestLogic_fk_subset_unique (9.54s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.66s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=25</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-63793

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343533?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343533?buildTab=artifacts#/) on master @ [1fcef18003b8dec750ed636aa92d35ca10105878](https://github.com/cockroachdb/cockroach/commits/1fcef18003b8dec750ed636aa92d35ca10105878):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique3105048462
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[16:09:05] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19992/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[16:09:05] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique3105048462
--- FAIL: TestLogic_fk_subset_unique (10.99s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19992/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19992/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.80s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343543?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343543?buildTab=artifacts#/) on master @ [1fcef18003b8dec750ed636aa92d35ca10105878](https://github.com/cockroachdb/cockroach/commits/1fcef18003b8dec750ed636aa92d35ca10105878):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique231135598
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[17:09:14] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1505/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[17:09:14] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique231135598
--- FAIL: TestLogic_fk_subset_unique (17.65s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1505/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1505/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.52s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343559?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343559?buildTab=artifacts#/) on master @ [1fcef18003b8dec750ed636aa92d35ca10105878](https://github.com/cockroachdb/cockroach/commits/1fcef18003b8dec750ed636aa92d35ca10105878):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique3609991011
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[17:24:29] setting distsql_workmem='88023B';
[17:24:35] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1510/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[17:24:36] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique3609991011
--- FAIL: TestLogic_fk_subset_unique (7.88s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1510/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1510/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.55s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/21343542?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/21343542?buildTab=artifacts#/) on master @ [1fcef18003b8dec750ed636aa92d35ca10105878](https://github.com/cockroachdb/cockroach/commits/1fcef18003b8dec750ed636aa92d35ca10105878):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/f610d4eb19743c36c25e2b9c085b1574/logTestLogic_fk_subset_unique1474067431
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[18:03:07] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/10306/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[18:03:08] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/f610d4eb19743c36c25e2b9c085b1574/logTestLogic_fk_subset_unique1474067431
--- FAIL: TestLogic_fk_subset_unique (14.05s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/10306/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/10306/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.96s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://mesolite.cluster.engflow.com/invocations/default/56c2d678-c9ed-4c90-99c0-fea49443f978?testReportRun=1&testReportShard=25&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy9sb2NhbC12ZWMtb2ZmOmxvY2FsLXZlYy1vZmZfdGVzdA==) on master @ [1fcef18003b8dec750ed636aa92d35ca10105878](https://github.com/cockroachdb/cockroach/commits/1fcef18003b8dec750ed636aa92d35ca10105878):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: outputs.zip/logTestLogic_fk_subset_unique401532491
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[04:27:56] setting distsql_workmem='63377B';
[04:28:05] --- done: /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[04:28:05] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: outputs.zip/logTestLogic_fk_subset_unique401532491
--- FAIL: TestLogic_fk_subset_unique (9.45s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.55s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=25</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343731?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343731?buildTab=artifacts#/) on master @ [1fcef18003b8dec750ed636aa92d35ca10105878](https://github.com/cockroachdb/cockroach/commits/1fcef18003b8dec750ed636aa92d35ca10105878):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique2601545512
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[06:47:26] setting distsql_workmem='50253B';
[06:47:33] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19653/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[06:47:34] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique2601545512
--- FAIL: TestLogic_fk_subset_unique (8.28s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19653/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19653/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.59s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343774?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343774?buildTab=artifacts#/) on master @ [152c95d20a52357c31375d9464f24d54a7482a94](https://github.com/cockroachdb/cockroach/commits/152c95d20a52357c31375d9464f24d54a7482a94):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique202782029
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[07:42:37] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19660/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[07:42:37] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique202782029
--- FAIL: TestLogic_fk_subset_unique (9.53s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19660/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/19660/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.78s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343798?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343798?buildTab=artifacts#/) on master @ [152c95d20a52357c31375d9464f24d54a7482a94](https://github.com/cockroachdb/cockroach/commits/152c95d20a52357c31375d9464f24d54a7482a94):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique1157123163
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[09:02:35] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1511/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[09:02:36] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique1157123163
--- FAIL: TestLogic_fk_subset_unique (7.92s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1511/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1511/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.55s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343814?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343814?buildTab=artifacts#/) on master @ [152c95d20a52357c31375d9464f24d54a7482a94](https://github.com/cockroachdb/cockroach/commits/152c95d20a52357c31375d9464f24d54a7482a94):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique556165995
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[09:18:39] setting distsql_workmem='63460B';
[09:18:45] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1530/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[09:18:46] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique556165995
--- FAIL: TestLogic_fk_subset_unique (6.97s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1530/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1530/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.45s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/21343783?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/21343783?buildTab=artifacts#/) on master @ [152c95d20a52357c31375d9464f24d54a7482a94](https://github.com/cockroachdb/cockroach/commits/152c95d20a52357c31375d9464f24d54a7482a94):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/f610d4eb19743c36c25e2b9c085b1574/logTestLogic_fk_subset_unique3477103937
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[10:38:56] setting distsql_workmem='83223B';
[10:39:06] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3803/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[10:39:07] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/f610d4eb19743c36c25e2b9c085b1574/logTestLogic_fk_subset_unique3477103937
--- FAIL: TestLogic_fk_subset_unique (12.20s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3803/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3803/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.87s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343868?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343868?buildTab=artifacts#/) on master @ [725a6009f2ace3de97d595db0242878726c53053](https://github.com/cockroachdb/cockroach/commits/725a6009f2ace3de97d595db0242878726c53053):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique3000193709
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[11:17:29] setting distsql_workmem='80112B';
[11:17:38] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20177/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[11:17:38] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique3000193709
--- FAIL: TestLogic_fk_subset_unique (9.51s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20177/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/20177/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.66s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://mesolite.cluster.engflow.com/invocations/default/096cebed-30b3-400c-8ecf-4472092d41ec?testReportRun=1&testReportShard=25&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy9sb2NhbC12ZWMtb2ZmOmxvY2FsLXZlYy1vZmZfdGVzdA==) on master @ [152c95d20a52357c31375d9464f24d54a7482a94](https://github.com/cockroachdb/cockroach/commits/152c95d20a52357c31375d9464f24d54a7482a94):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: outputs.zip/logTestLogic_fk_subset_unique2879814742
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[10:34:23] --- done: /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[10:34:26] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: outputs.zip/logTestLogic_fk_subset_unique2879814742
--- FAIL: TestLogic_fk_subset_unique (86.20s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
[10:34:22] --- progress: /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique: 135 statements
    logic.go:3409: 
         
        /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (6.26s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=25</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343924?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343924?buildTab=artifacts#/) on master @ [725a6009f2ace3de97d595db0242878726c53053](https://github.com/cockroachdb/cockroach/commits/725a6009f2ace3de97d595db0242878726c53053):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique3229675431
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[12:18:52] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1505/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[12:18:53] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique3229675431
--- FAIL: TestLogic_fk_subset_unique (10.07s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1505/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1505/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.44s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343940?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21343940?buildTab=artifacts#/) on master @ [725a6009f2ace3de97d595db0242878726c53053](https://github.com/cockroachdb/cockroach/commits/725a6009f2ace3de97d595db0242878726c53053):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique403109150
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[12:35:57] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1527/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[12:35:58] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/13e35ae630aaa8edd1b312b8bbf8c61f/logTestLogic_fk_subset_unique403109150
--- FAIL: TestLogic_fk_subset_unique (7.79s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1527/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1527/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.51s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21343591?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_SqlLogicTestHighVModuleNightlyBazel/21343591?buildTab=artifacts#/) on master @ [1fcef18003b8dec750ed636aa92d35ca10105878](https://github.com/cockroachdb/cockroach/commits/1fcef18003b8dec750ed636aa92d35ca10105878):


```
      ndirty: 0 huge, 0 nonhuge 
      nretained: 0 huge, 0 nonhuge 
  In empty slabs:
      npageslabs: 0 huge, 0 nonhuge
      nactive: 0 huge, 0 nonhuge 
      ndirty: 0 huge, 0 nonhuge 
      nretained: 0 huge, 0 nonhuge 

                size ind npageslabs_huge    nactive_huge     ndirty_huge  npageslabs_nonhuge     nactive_nonhuge      ndirty_nonhuge   nretained_nonhuge
                     ---
--- End jemalloc statistics ---
[12:02:48] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd7bae8917ba48935e4db8ce01527a76/logTestLogic_fk_subset_unique2202784484
--- FAIL: TestLogic_fk_subset_unique (52.92s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
[12:02:42] CREATE TABLE drop_ref_parent (a INT PRIMARY KEY, b INT);
[12:02:42] rewrote:
CREATE TABLE drop_ref_parent (a INT8 PRIMARY KEY, b INT8, FAMILY (b, a));


[12:02:42] CREATE TABLE drop_ref_child (
  a INT, b INT,
  CONSTRAINT drop_ref_fk FOREIGN KEY (a, b) REFERENCES drop_ref_parent (a, b)
);
[12:02:42] rewrote:
CREATE TABLE drop_ref_child (a INT8, b INT8, CONSTRAINT drop_ref_fk FOREIGN KEY (a, b) REFERENCES drop_ref_parent (a, b), FAMILY (a, b));


    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
[12:02:43] ALTER TABLE drop_ref_parent DROP COLUMN b;
[12:02:43] ALTER TABLE drop_ref_parent DROP COLUMN b CASCADE;
[12:02:45] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4227/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique: 135 statements
[12:02:45] SHOW CONSTRAINTS FROM drop_ref_child;
[12:02:45] 	-- FAIL
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4227/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
[12:02:45] 	-- FAIL
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4227/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (3.40s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_fk_subset_unique [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/21343877?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/21343877?buildTab=artifacts#/) on master @ [725a6009f2ace3de97d595db0242878726c53053](https://github.com/cockroachdb/cockroach/commits/725a6009f2ace3de97d595db0242878726c53053):


```
=== RUN   TestLogic_fk_subset_unique
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/f610d4eb19743c36c25e2b9c085b1574/logTestLogic_fk_subset_unique2931462815
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:150: cluster virtualization disabled due to issue: #156124 (expected label: C-bug)
[14:07:06] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3074/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique with config local-vec-off: 136 tests, 2 failures
[14:07:06] --- total progress: 136 statements
--- total: 136 tests, 2 failures
    logic.go:5004: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/f610d4eb19743c36c25e2b9c085b1574/logTestLogic_fk_subset_unique2931462815
--- FAIL: TestLogic_fk_subset_unique (9.95s)
=== RUN   TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique
    logictestbase.go:575: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports local-legacy-schema-changer (no issue given)
    logic.go:3409: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3074/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:719: SHOW CONSTRAINTS FROM drop_ref_child
        expected:
            drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            
        but found (query options: "nosort") :
            drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
            drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
            
    logic.go:2530: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3074/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/fk_subset_unique:724: too many errors encountered, skipping the rest of the input
--- FAIL: TestLogic_fk_subset_unique/subset_unique_drop_referenced_column_not_in_backing_unique (0.59s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_fk_subset_unique.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/logictest/testdata/logic_test/fk_subset_unique
+++ b/pkg/sql/logictest/testdata/logic_test/fk_subset_unique
@@ -718,7 +718,8 @@ ALTER TABLE drop_ref_parent DROP COLUMN b CASCADE
 query TTTTB nosort
 SHOW CONSTRAINTS FROM drop_ref_child
 ----
-drop_ref_child  drop_ref_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
+drop_ref_child  drop_ref_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
+drop_ref_child  drop_ref_child_rowid_not_null  NOT NULL     NOT NULL                 true
 
 statement ok
 DROP TABLE drop_ref_child, drop_ref_parent
@@ -761,7 +762,8 @@ ALTER TABLE uwi_inbound_parent DROP COLUMN d CASCADE
 query TTTTB nosort
 SHOW CONSTRAINTS FROM uwi_inbound_child
 ----
-uwi_inbound_child  uwi_inbound_child_pkey  PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
+uwi_inbound_child  uwi_inbound_child_pkey            PRIMARY KEY  PRIMARY KEY (rowid ASC)  true
+uwi_inbound_child  uwi_inbound_child_rowid_not_null  NOT NULL     NOT NULL                 true
 
 statement ok
 DROP TABLE uwi_inbound_child;

